src/content/developers/docs/scaling/optimistic-rollups/index.md

```diff
@@ -1,46 +1,254 @@
+
+### Censorship resistance {#censorship-resistance}
+
+Optimistic rollups also rely on Ethereum for censorship resistance. In an optimistic rollup a centralized entity (the operator) is responsible for processing transactions and submitting rollup blocks to Ethereum. This has some implications: 
+
+- Rollup operators can censor users by going offline, refusing to produce blocks, or submit user transactions to L1. 
+
+- Rollup operators can prevent users from withdrawing funds deposited in the rollup contract by withholding state data necessary to Merkle proofs of ownership. Withholding state data can also conceal the rollup’s state from users and prevent them from interacting with the rollup.  
+
+Optimistic rollups solve this problem by forcing operators to publish data associated with state updates on Ethereum. Publishing rollup data on-chain has the following benefits:
+
+- If an optimistic rollup operator goes offline or stops producing transaction batches, another node can use available data to reproduce the rollup’s last state and continue block production. 
+
+- Users can use transaction data to create Merkle proofs proving ownership of funds and withdraw their assets from the rollup. 
+
+### Settlement {#settlement}
+
+Another role Ethereum plays in the context of optimistic rollups is that of a settlement layer. A settlement layer anchors the entire blockchain ecosystem, establishes security, and provides objective finality if a dispute occurs on another chain (optimistic rollups in this case) that requires arbitration. 
+
+Ethereum Mainnet provides a hub for optimistic rollups to verify fraud proofs and resolve disputes. Moreover, transactions conducted on the rollup are only final *after* the rollup block is accepted on Ethereum. Once a rollup transaction is committed to Ethereum’s base layer, it cannot be rolled back (except in the highly unlikely case of a chain reorganization).
+
+## How optimistic rollups work {#how-optimistic-rollups-work} 
 
-Optimistic rollups sit in parallel to the main Ethereum chain on layer 2. They can offer improvements in scalability because they don't do any computation by default. Instead, after a transaction, they propose the new state to Mainnet or "notarise" the transaction.
+### Transaction execution and aggregation {#transaction-execution-and-aggregation}
 
-With Optimistic rollups, transactions are written to the main Ethereum chain as `calldata`, optimising them further by reducing the gas cost.
+Users submit transactions to “operators”, nodes responsible for processing transactions on the optimistic rollup. Also known as a “validator” or “aggregator”, the operator aggregates transactions, compresses the underlying data, and publishes the block on Ethereum. 
```

Comment:
This makes it sound there are multiple transaction sources at the same time, which is incorrect (at least for Optimism, I don't know Arbitrum well enough).

How about

> Users submit their transactions to nodes that forward them to the sequencer (there is only one sequencer at a time). The sequencer then aggregates the transactions, compresses them, and publishes the block on Ethereum.

A node does not need to be a validator, and a validator does not need to provide node services.